project_euler/problem_10/sol3.py

```diff
@@ -1,20 +1,64 @@
-#https://projecteuler.net/problem=10
-#using Sieve_of_Eratosthenes
+"""
+https://projecteuler.net/problem=10
+
+Problem Statement:
+The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.
+
+Find the sum of all the primes below two million.
+
+using Sieve_of_Eratosthenes : 
+
+The sieve of Eratosthenes is one of the most efficient ways to find all primes smaller than n when n is smaller than 10 million
+"""
+
+try:
+    raw_input  # Python 2
+except NameError:
+    raw_input = input  # Python 3
+
+try:
+    xrange  # Python 2
+except NameError:
+    xrange = range  # Python 3
```

Comment:
Sorry to be a pain but...  Let's simplify and just use __input()__ and __range()__ because this repo no longer supports legacy Python.